threads/src/main/java/org/jboss/as/threads/ThreadsParser.java

```diff
@@ -512,6 +513,12 @@ public String parseUnboundedQueueThreadPool(final XMLExtendedStreamReader reader
                     foundMaxThreads = true;
                     break;
                 }
+                case CORE_THREADS: {
```

Comment:
never mind, it looks like it is bug in old version of parser.
2.0 parser handles is correctly as is.